src/style.css

```diff
@@ -0,0 +1,106 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+@media (min-width: 600px) {
+  .main {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+}
+
+.block {
+  height: 300px;
+  min-width: 300px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-grow: 1;
+  font-size: 100px;
+  color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+@media (min-width: 1200px) {
+  .block {
+    min-width: 400px;
+  }
+}
+
+.one {
+  background-color: rgb(255, 0, 0);
+}
+
+.two {
+  background-color: rgb(204, 0, 0);
+}
+
+.three {
+  background-color: rgb(153, 0, 0);
+}
+
+@media (min-width: 600px) {
+  .three {
+    order: 2;
+  }
+}
+
+@media (min-width: 900px) {
+  .three {
+    order: 1;
+  }
+}
+
+.four {
+  background-color: rgb(102, 0, 0);
+}
+
+@media (min-width: 600px) {
```

Comment:
Don't duplicate your media queries. You may write once 
`@media (min-width: 600px) { }`
and write all relevant to this width styles there (not write query many times)